src/apps/aave/positions.ts

```diff
@@ -188,10 +188,31 @@ const hook: PositionsHook = {
 
         // Include a token if the user has a balance
         // or if no user data is available (when querying all positions)
+
+        function isGreaterThanZero(
+          input:
+            | number
+            | readonly {
+                scaledATokenBalance: bigint
+                scaledVariableDebt: bigint
+              }[],
```

Comment:
this is a strange type. Is it safe to set `useAToken` and `useVariableDebt` both to true if it is a non zero number? Do we know when this would be a number and when an object? If so can we document it?

Reply:
Honestly, I couldn't find a single case in their contracts where this would resolve to an object. Maybe its a part of future plans?